ext/standard/tests/file/005_variation.phpt

```diff
@@ -35,15 +35,15 @@ if (getenv("SKIP_SLOW_TESTS")) {
 */
 function stat_fn( $filename ) {
   echo "-- File access time is => ";
-  print( @date( 'Y:M:D:H:i:s', fileatime($filename) ) )."\n";
+  print( $acessTime = @date( 'Y:M:D:H:i:s', fileatime($filename) ) )."\n";
   clearstatcache();
   echo "-- File modification time is => ";
-  print( @date( 'Y:M:D:H:i:s', filemtime($filename) ) )."\n";
+  print( $modificationTime = @date( 'Y:M:D:H:i:s', filemtime($filename) ) )."\n";
   clearstatcache();
   echo "-- inode change time is => ";
-  print( @date( 'Y:M:D:H:i:s', filectime($filename) ) )."\n";
+  print( $inodeTime = @date( 'Y:M:D:H:i:s', filectime($filename) ) )."\n";
   clearstatcache();
-
+  var_dump($acessTime <= $modificationTime && $acessTime <= $inodeTime);
```

Comment:
I'm not sure this check makes sense and it seems pretty unreliable (access time updates are sensitive to mount configuration.)

Reply:
I've removed than and just reduce the `sleep`s.